jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/AddRemoveLogGroupTable.kt

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+import com.intellij.execution.util.ListTableWithButtons
+import com.intellij.openapi.project.Project
+import com.intellij.util.ui.ListTableModel
+import software.aws.toolkits.resources.message
+import javax.swing.table.TableCellEditor
+
+class AddRemoveLogGroupTable(project: Project) : AddRemoveLogs<SelectedLogGroups> (emptyTableMainText = "No log groups found",
+    addNewEntryText = "Add log groups") {
+    override fun cloneElement(variable: SelectedLogGroups): SelectedLogGroups = variable.copy()
+    override fun createElement(): SelectedLogGroups = SelectedLogGroups()
+    fun getSelLogGroups(): List<SelectedLogGroups> = elements.toList()
+    override fun createListModel(): ListTableModel<*> = ListTableModel<SelectedLogGroups>(
+        StringColInfo(
+            message("cloudwatch.logs.loggq"),
+            { it.log_groups },
+            { mapping, value -> mapping.log_groups = value }
+        )
+    )
+
+    private inner class StringColInfo(
```

Comment:
This stuff should be outside of this class in its own file

Reply:
Displaying a sample table. Added the TODO section in the class

Reply:
This will be used in the real table though?

Reply:
Yes, it will be, with a few changes. Although I am not sure if I should create a separate file for it since it will have ListTableWithButtons as a parent class and some functions which are already used in AddRemoveLogGroupTable will have to be rewritten.